API/raw_data.py

```diff
@@ -330,3 +331,210 @@ def watch_s3_upload(url: str, path: str) -> None:
         logging.debug(
             "File is uploaded at %s , flushing out from %s", url, path)
         os.unlink(path)
+
+
+@router.post("/current-snapshot/")
+@limiter.limit(f"{export_rate_limit}/minute")
```

Comment:
We should add this limit in the doc.

[Maybe this stackoverflow could help](https://stackoverflow.com/a/65842969).

Reply:
I will add limit info on doc and raise formatted error response